### ISSUE TITLE
logout and login urls

### ISSUE THREAD

**Issue by gcetusic** [VorskiImagineering/django-C3PO#2]
The  README doesn't say you have to set the LOGIN_URL and LOGOUT_URL in the C3PO dict. Also, if we include the urls in the project urls.py as recommended in README it can't resolve the logout and login urls and we get something like this:

NoReverseMatch at /c3po/
Reverse for '/social/login/google-oauth2/' with arguments '()' and keyword arguments '{}' not found.

This particular problem happens when you access the url without logging in first. This isn't that much of a problem because a user shouldn't access c3po without being logged in but this is connected with another problem once you do log in, another error pops up because of {% url logout_url %} in base.html.

Reverse for '/accounts/logout/' with arguments '()' and keyword arguments '{}' not found.

Note that this entry DOES exist in urls.py:

urlpatterns = patterns('',
    url(r'^admin/', include(admin.site.urls)),

```
url(r'^accounts/logout/$', 'django.contrib.auth.views.logout',
    {'next_page': '/'}, name='logout'),
.
.
.
url(r'^c3po/', include('django_c3po.urls'))
```

)
urlpatterns += staticfiles_urlpatterns()

Changing to {% url 'logout' %} solves the problem because the url is read from urls.py. Note that the quotes around logout are required in Django 1.5. Any ideas?

**Comment by stefan8** [comment]
We need to complete readme with this two new parameters. I think you put the full url path into config and this is why it's not working. These parameters are passed to reverse / url tag. For example:

```
...
'LOGIN_URL': 'auth_login',
'LOGOUT_URL': 'auth_logout', # or just 'logout' in your case
...
```

**Comment by gcetusic** [comment]
What's the purpose of the login_url? 
Because if I set it to 'LOGIN_URL': 'web_landing_page' and have my urls set to

```
url(r'^social/', include('social_auth.urls')),
url(r'^c3po/', include('django_c3po.urls')),
(r'^jsi18n/$', 'django.views.i18n.javascript_catalog'),
url(r'^api/v1/', include('api.urls')),
url(r'^', include('web.urls')),
```

with the last url representing the urls.py that has "url(r'^$', 'landing_page', name='web_landing_page')" I can't access /c3po/ because it always redirects me to /

My login screen uses oauth and LOGIN_URL in settings.py is "/social/login/google-oauth2/" which c3po can't resolve in its views.py so I can't use it directly for some reason.

**Comment by stefan8** [comment]
You can pass view path, or url name (https://docs.djangoproject.com/en/dev/ref/urlresolvers/#django.core.urlresolvers.reverse). You are being redirected because your are not logged in or don't have needed permission. Probably fallback to settings.LOGIN_URL could be better solution.

**Comment by gcetusic** [comment]
yes, that was the problem, the view needed 'django_c3po.can_translate' and I didn't have a superuser...Thanks
